src/utils/AddressUtils.ts

```diff
@@ -275,12 +258,22 @@ export class SvmAddress extends Address {
     }
 
     super(rawAddress);
-    // @dev required for TS to compile with `noUnusedLocals` rule
-    this._brandSvmAddress;
+    this._type; // tsc noUnusedLocals appeasement.
   }
 
   static validate(rawAddress: Uint8Array): boolean {
-    return rawAddress.length === 32;
+    // Deliberately invalidate SVM addresses w/ the upper 12 bytes zeroed. These addresses are technically valid
+    // but highly improbable and are much more likely to be a mistaken interpretation of an EVM address. Err on
+    // the side of caution for the time being. Exception: Permit the zero address (i.e. for exclusiverRelayer).
+    // @todo Add test for this.
```

Comment:
To be fixed prior to merging.

Reply:
https://github.com/across-protocol/sdk/pull/1101/commits/fc55febb240c67c6982caee50d57c5c665b8d27d